Source/WebCore/page/scrolling/ScrollingStateNode.cpp

```diff
@@ -173,6 +173,18 @@ void ScrollingStateNode::setLayer(const LayerRepresentation& layerRepresentation
     setPropertyChanged(Property::Layer);
 }
 
+#if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
+void ScrollingStateNode::setInteractionRegionsLayer(const LayerRepresentation& layerRepresentation)
+{
+    if (layerRepresentation == m_interactionRegionsLayer)
+        return;
+
+    m_interactionRegionsLayer = layerRepresentation;
+
+    // Depends on other property changed flags.
```

Comment:
I'd like this to be more specific. In particular, the interactions region is special, in terms of dirty bits, because it doesn't have its own; you're piggybacking on `ScrollingStateNodeProperty::Layer` I think?

Reply:
and `ScrollContainerLayer` for the `ScrollingTreeScrollingNodeDelegateIOS`